tests/unit/ContinuationIn.spec.ts

```diff
@@ -0,0 +1,531 @@
+/* eslint-disable max-len */
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { createLocalVue, mount } from '@vue/test-utils'
+import { createPinia, setActivePinia } from 'pinia'
+import mockRouter from './mockRouter'
+import VueRouter from 'vue-router'
+import { useBusinessStore, useFilingHistoryListStore, useRootStore } from '@/stores'
+import { DateUtilities, LegalServices } from '@/services'
+import { FilingTypes } from '@bcrs-shared-components/enums'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+import TodoList from '@/components/Dashboard/TodoList.vue'
+import PendingList from '@/components/Dashboard/PendingList.vue'
+import FilingHistoryList from '@/components/Dashboard/FilingHistoryList.vue'
+import DocumentsList from '@/components/Dashboard/FilingHistoryList/DocumentsList.vue'
+import { ContactInfo } from '@/components/common'
+import { FilingStatus } from '@/enums'
+
+const vuetify = new Vuetify()
+const localVue = createLocalVue()
+const router = mockRouter.mock()
+
+Vue.use(Vuetify)
+localVue.use(VueRouter)
+
+setActivePinia(createPinia())
+const businessStore = useBusinessStore()
+const filingHistoryStore = useFilingHistoryListStore()
+const rootStore = useRootStore()
+
+describe('Continuation In - in To Do list', () => {
+  it('displays a draft continuation application (with NR)', async () => {
+    // init session storage + store
+    sessionStorage.setItem('TEMP_REG_NUMBER', 'T1234567')
+    rootStore.setCurrentJsDate(new Date('2024-07-24T06:59:00.000Z'))
```

Comment:
I picked this weird-looking time to match the NR expiration date on line 58 so that the unit tests would report "8 days" locally and in CI environment (which doesn't support UTC offsets very well). Without this, locally the NR expired in "7 days" but in CI it expired in 8 days :(